docs/ovms_quickstart.md

```diff
@@ -0,0 +1,170 @@
+# OpenVINO&trade; Model Server Quickstart
+
+## Deploying Model Server
+
+Model Server can deployed using any of the following options:
+1. Download a pre-built container image from Docker Hub
+2. Build a container image with a Dockerfile
+3. Build the application locally from source 
+
+Each approach is described below.
+
+### Pre-built Docker Container
+
+A pre-built container with OpenVINO&trade; Model Server is available for download on [Docker Hub](https://hub.docker.com/r/openvino/ubuntu18_model_server). The image
+is based on Ubuntu 18.04 and contains all required components needed to perform inference. 
+
+To start using this image, you can pull it with the following command:
+
+```bash
+docker pull openvino/ubuntu18_model_server
+```
+
+### Build with a Dockerfile
+
+Three Dockerfiles are provided that can be used to build a container image locally. There is also a Makefile to simplify building an image. The Dockerfiles are listed below:
+
+#### 1. [Ubuntu 18 Dockerfile](https://github.com/openvinotoolkit/model_server/blob/master/Dockerfile)
+
+To build this image use following command from the `model_server` directory:
+
+```bash
+make docker_build_apt_ubuntu
+```
+
+#### 2. [Clear Linux Dockerfile](https://github.com/openvinotoolkit/model_server/blob/master/Dockerfile_clearlinux)

```

Comment:
I think we should drop clearlinux for now. It is lagging behind with the version 2019.3

Reply:
I agree. We should not be stuck at 2019.3. Let's drop it.

Reply:
Removed.